packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/connection/DatabaseBuilderState.ts

```diff
@@ -350,12 +354,6 @@ export class DatabaseSchemaExplorerState {
   }
 
   get resolveDatabasePackageAndName(): [string, string] {
-    if (!this.isCreatingNewDatabase) {
```

Comment:
why are we removing this. if we are not creating a new db then the package should be there.

Reply:
To support uses to click the 'back' button and this is still editable

Reply:
did you look into how it would affect where its already used? dont think thats a valid reason, the change should make sense in isolation of where its used but how it makes sense in `DatabaseBuilderState`

Reply:
For the original feature, the difference would be the target path is always editable, which kind of makes senes to me..
if users want to create a new db with a new path name, now this is doable.
If users just want to update the original db, then just keep the value as what it is before